website/src/components/Chat/ChatConversation.tsx

```diff
@@ -305,6 +313,33 @@ export const ChatConversation = memo(function ChatConversation({ chatId, getConf
             {t("queue_info", queueInfo)}
           </Badge>
         )}
+        {pluginIntermediateResponse && pluginIntermediateResponse.currentPluginThought && (
+          <Box
+            position="absolute"
+            bottom="0"
+            left="50%"
+            transform="translate(-50%)"
+            display="flex"
+            flexDirection="row"
+            gap="1"
+            justifyContent="center"
+            alignItems="center"
+          >
+            <Box
+              bg="purple.700"
+              color="white"
+              px="2"
+              py="2.5px"
+              borderRadius="8px"
+              maxWidth="50vw"
+              fontSize="11"
+              fontWeight="bold"
+              isTruncated
+            >
+              {pluginIntermediateResponse.currentPluginThought}
+            </Box>
+          </Box>
+        )}
```

Comment:
Have you check UI in the light mode?

Reply:
I didn't.  I will check it soon

Reply:
<img width="971" alt="Screenshot 2023-05-28 at 15 00 00" src="https://github.com/LAION-AI/Open-Assistant/assets/13547364/d53be7f4-368c-4d12-bdeb-8e3f710bbfab">

Reply:
Yeah, that look good.